docs/hsi.md

```diff
@@ -144,15 +144,21 @@ UEFI Secure boot is a verification mechanism for ensuring that code launched by
 
 Secure Boot requires that each binary loaded at boot is validated against trusted certificates.
 
-Possible results:
+**Impact:** When Secure Boot is not enabled any EFI binary can be run at startup, which gives the local attacker full access to your hardware.
```

Comment:
Why they gotta be local? UEFI has network stacks these days...

Reply:
How about adding `If UEFI networking is enabled then this widens the attack surface to remote attackers too.` -- or I can just remove the `local` word...

Reply:
Some bios vendors might not let you disable it.
I say remove local.

Reply:
Done, thanks.